packages/@react-stately/disclosure/docs/useDisclosureGroupState.mdx

```diff
@@ -0,0 +1,228 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-stately/disclosure';
+import {ClassAPI, HeaderInfo, TypeContext, FunctionAPI, TypeLink, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-stately/disclosure/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+
+---
+category: Navigation
+keywords: [disclosure, collapse, expand, state]
+preRelease: alpha
+---
+
+# useDisclosureGroupState
+
+<PageDescription>{docs.exports.useDisclosureGroupState.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useDisclosureGroupState']} />
+
+## API
+
+<FunctionAPI function={docs.exports.useDisclosureGroupState} links={docs.links} />
+
+## Interface
+
+<ClassAPI links={docs.links} class={docs.links[docs.exports.useDisclosureGroupState.return.id]} />
+
+## Example
+
+The following example shows how to create a `DisclosureGroup` component with the `useDisclosureGroupState` hook. We'll also create a `Disclosure` component that uses the `DisclosureGroupState` context for managing its state.
```

Comment:
we don't typically do full examples in stately because they require aria hooks
I think we should move this to the `useDisclosure` page and combine GroupState into the `useDisclosureState` page instead of having a separate page